topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -1,14 +1,13 @@
 ---
 layout: tutorial_hands_on
-enable: false
 
 title: 'Mass spectrometry: LC-MS preprocessing - advanced'
```

Comment:
Instead of calling it advanced, you could give it two „hats“ by adding: level: Intermediate

Reply:
Indeed! Thanks.
I took this opportunity to change the name to "Mass spectrometry: LC-MS preprocessing with XCMS" to make it more precise. 
See  8a02dcc